flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/functions/sql/FlinkSqlOperatorTable.java

```diff
@@ -1148,9 +1162,21 @@ public SqlMonotonicity getMonotonicity(SqlOperatorBinding call) {
 	public static final SqlFunction PI = SqlStdOperatorTable.PI;
 	public static final SqlFunction RAND = SqlStdOperatorTable.RAND;
 	public static final SqlFunction RAND_INTEGER = SqlStdOperatorTable.RAND_INTEGER;
+	public static final SqlFunction TRUNCATE = SqlStdOperatorTable.TRUNCATE;
+
+	// TIME FUNCTIONS
+	public static final SqlFunction YEAR = SqlStdOperatorTable.YEAR;
```

Comment:
what's these time functions used for?

Reply:
supports expression like `EXTRACT(YEAR FROM f)`

Reply:
IIRC, this expression already supported with previous calcite version

Reply:
before 1.20, `YEAR(f)` will be converted to `EXTRACT(YEAR FROM f)` in sql parser, and `YEAR` is treated as a `SqlIntervalQualifier` and will not lookup operator table . while in 1.20, the parser's behavior is changed, `YEAR(f)` is a function call, and will lookup operator table.